--- origsrc/libopensync-plugin-google-calendar-0.22/src/Makefile.am	2007-03-27 11:18:21.000000000 -0500
+++ src/libopensync-plugin-google-calendar-0.22/src/Makefile.am	2008-05-09 09:51:11.296875000 -0500
@@ -12,9 +12,7 @@
 
 plugin_LTLIBRARIES = gcalendar.la
 
-libexec_PROGRAMS = google-cal-helper
-
-google_cal_helper_SOURCES = google-cal-helper.py
+libexec_SCRIPTS = google-cal-helper
 
 gcalendar_la_SOURCES = gcalendar.c
 gcalendar_la_CFLAGS = -DGCAL_HELPER=\"$(libexecdir)/google-cal-helper\"
